scripts/assets/options.go.templ

```diff
@@ -109,6 +109,8 @@ func UpdateFieldsForJSON(flags *pflag.FlagSet) (*serverSettingsForJSON, error) {
 		}
 	<% } %>
 	if changedSomething {
+	  specifiedSettings.Version = <%- settingsVersion %>
```

Comment:
`settingsVersion` isn't being updated ... it's the current version constant that the code-generator passes into the structure, and the API requires us to pass in.

And keep in mind I use two structures ... one with allocated space for each field, and the second with pointers so the payload only contains the changed fields.

Maybe I don't understand your comment here.

Reply:
The full context is:
```
	if changedSomething {
	  specifiedSettings.Version = <%- settingsVersion %>
	  specifiedSettingsForJSON.Version = &specifiedSettings.Version;
		return &specifiedSettingsForJSON, nil
	}
```
I understand the second set (`specifiedSettingsForJSON.Version = `…) because that gets returned, but we _also_ set `specifiedSettings.Version` and _don't_ return it. That seems like it's either ① pointless work, or ② spookily modifying a global and using the result elsewhere (which is even scarier).

(Now that I see it outside the diff view, I also see that the indent is messed up here…)

Reply:
Because go went with tabs and I can't use `gofmt` on a template file.